### PR TITLE
Add new syslib warnings

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #23856 

[Internal preview](https://review.docs.microsoft.com/en-us/dotnet/core/compatibility/syslib-obsoletions?branch=pr-en-us-23915#reference).

### CODE REVIEW

**Review by jeffhandley — APPROVED** [review]
Looks good; thanks!  If you want to add `SYSLIB0017` and `SYSLIB0018` in while you're at it, those are included in https://github.com/dotnet/runtime/pull/50941.

File: docs/core/compatibility/syslib-warnings/syslib0013.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: SYSLIB0013 warning
```
Comment (Youssef1313) [inline docs/core/compatibility/syslib-warnings/syslib0013.md:2]: `https://aka.ms/dotnet-warnings/SYSLIB0013` may need to get updated to this article after it's published. (Same applies to other obsoletion warnings.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

File: docs/core/compatibility/syslib-warnings/syslib0013.md (hunk 1 above)
Comment (gewarren) [reply docs/core/compatibility/syslib-warnings/syslib0013.md:2]: @jeffhandley I don't seem to have access to these aka.ms links. Can you update for 13-16, 19-20?

Comment (gewarren) [reply docs/core/compatibility/syslib-warnings/syslib0013.md:2]: I will publish these live now.

Comment (jeffhandley) [reply docs/core/compatibility/syslib-warnings/syslib0013.md:2]: I went ahead and created the aka.ms links for SYSLIB0017 through SYSLIB0025. I don't (yet) have access to edit the existing ones, but I submitted a request to gain access.

**Pull request merged**